autogpt/app.py

```diff
@@ -181,6 +182,9 @@ def execute_command(command_name: str, arguments):
                 )
         elif command_name == "generate_image":
             return generate_image(arguments["prompt"])
+        elif command_name == "send_email":
+            if CFG.use_gmail:
```

Comment:
Wont this add some unexpected behavior? Shouldn't we return something "send_email command not available" if email is not configured?

Reply:
yea you are right. i forgot to add a else part